serine.galstyan/repo/Lesson_3/exercise_07/exercise_07.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/evn python3
+
+entered_kgs = input("Enter how many kilograms should convert:")
```

Comment:
CST: "Enter how many kilograms should be converted" will be grammatically more correct.